app/javascript/components/rooms/room/Room.jsx

```diff
@@ -58,6 +59,10 @@ export default function Room() {
   return (
     <>
       <div className="wide-white">
+        <Helmet>
```

Comment:
This is really nice !
We could create a component called Title with a children prop as the value maybe to improve the syntax?
```
<Title> My Title !</Title>
```